packages/config/config/devices/0x0234/zif5020_2.0.json

```diff
@@ -0,0 +1,208 @@
+{
+	"manufacturer": "Logic Group",
+	"manufacturerId": "0x0234",
+	"label": "ZIF5020",
+	"description": "DIN Rail Interface",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x010d"
+		}
+	],
+	"firmwareVersion": {
+		"min": "2.0",
+		"max": "255.255"
```

Comment:
We don't split files like this anymore for different firmware versions. Please combine them and use conditional parameters as necessary. In the documentation.

Reply:
Its basically 2 entirely different devices but with the same ID annoyingly. The lhc5020 is the original and then i made the zif5020 because thats its new name. So the lhc5020 is basically obsolete but i dont want to destroy others installation

Reply:
Ok